workspaces/sec-literal/test/isStringBase64.spec.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @stylistic/max-len */
+// Import Node.js Dependencies
+import { test } from "node:test";
+import assert from "node:assert/strict";
+
+// Import Internal Dependencies
+import { isStringBase64 } from "../src/utils.js";
+
+test("isBase64", function isBase64() {
```

Comment:
I just would have split it in different test cases. Just in case of regression we could see which one fails

Reply:
That's the original test-suite of the package ^^, i'm quite ok with it since anyway stacktrace show which assertion fail.